README.md

```diff
@@ -115,3 +115,15 @@ tools/src/eclipse/apiman-eclipse-formatter.xml
 
 You should be able to import that guy straight into Eclipse by going to
 *Window->Preferences :: Java/Code Style/Formatter*
+
+## Developer Portal
+
+Apiman also comes with an developer portal. There you can allow Developers to access your APIs. Developers can view and test your APIs to develope their own Apps.\
```

Comment:
couple typos:  should be "...with a developer portal..." and "...APIs to develop..."

Reply:
Fixed. Thanks for the hint :)